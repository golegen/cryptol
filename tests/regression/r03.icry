:set warnDefaulting=off
:m r03
:set show-examples=off
:sat nQueens:Solution 5
it.result
:prove (nQueensProve:Solution 4)
it.result
