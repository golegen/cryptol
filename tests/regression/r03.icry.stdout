Loading module Cryptol
Loading module r03
True
False
